Sources/InstantSearch/Hits/CollectionView/HitsCollectionViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  HitsCollectionViewController.swift
+//  DemoDirectory
+//
+//  Created by Vladislav Fitc on 29/07/2020.
+//  Copyright © 2020 Algolia. All rights reserved.
+//
+
+#if !InstantSearchCocoaPods
+import InstantSearchCore
+#endif
+#if canImport(UIKit) && (os(iOS) || os(tvOS) || os(macOS))
+import UIKit
+
+open class HitsCollectionViewController<CellConfigurator: CollectionViewCellConfigurable>: UICollectionViewController, UICollectionViewDelegateFlowLayout, HitsController {
```

Comment:
Do we want to document this now, or after with everything else once and for all?

Reply:
Good point, will add a brief code documentation.